server/package.json

```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "start": "ENV=prod node -r esm app.js",
     "dev": "source .env && ENV=dev nodemon -r esm app.js",
-    "start-w": "set ENV=prod && nodemon -r esm app.js",
-    "dev-w": "set ENV=dev && nodemon -r esm app.js",
+    "start-w": "set ENV=prod&& env && nodemon -r esm app.js",
```

Comment:
I needed to include '&& env'. Note that the I removed the space between prod and && intentionally because windows for some reason includes that as the variable value.